databricks/koalas/frame.py

```diff
@@ -8539,7 +8539,7 @@ def query(self, expr, inplace=False):
                 1  3  4
                 2  5  6
                 >>> num = 1
-                >>> df.map_in_pandas(lambda pdf: pdf.query('A > @num'))  # doctest: +SKIP
+                >>> df.map_in_pandas(lambda pdf: pdf.query('A > @num'))
```

Comment:
I'm wondering whether this example works with Spark.
I guess the `num` can't be seen from executors.
Actually, it throws an exception:

```py
>>> import databricks.koalas as ks
>>> df = ks.DataFrame([(1, 2), (3, 4), (5, 6)], columns=['A', 'B'])
>>> num = 1
>>> df.map_in_pandas(lambda pdf: pdf.query('A > @num'))
   A  B
1  3  4
2  5  6

>>> ks.options.compute.shortcut_limit = 1
>>> df.map_in_pandas(lambda pdf: pdf.query('A > @num'))
org.apache.spark.api.python.PythonException: Traceback (most recent call last):

...

pandas.core.computation.ops.UndefinedVariableError: local variable 'num' is not defined
```

Reply:
@ueshin 

yes, i also just found some problem with `map_in_pandas` for those kinda methods, that it only works properly when the `compute.shortcut_limit` is larger than the size of the data during i'm working at #1292 .

so i think this example for `query` is not illustrated properly, we better remove this example or adding the note that we can work around with `map_in_pandas` when only the size of the data is smaller than `compute.shortcut_limit`.

which way is more proper do you think??

Thanks for the comment :)

Reply:
@itholic can you fix the example? Also update "the exactly same syntax with pandas" in the docstring too

Reply:
okay, i'll start right away, thanks for the comment !